cogeo_mosaic/backends/memory.py

```diff
@@ -0,0 +1,62 @@
+"""cogeo-mosaic InMemory backend."""
+
+from typing import Dict, List, Tuple, Type
+
+import attr
+from morecantile import TileMatrixSet
+from rio_tiler.constants import WEB_MERCATOR_TMS
+from rio_tiler.io import BaseReader, COGReader
+
+from cogeo_mosaic.backends.base import (
+    BaseBackend,
+    mode_validator,
+    update_zoom_and_bounds,
```

Comment:
See this seems weird. Why not just put `mode_validator` and `update_zoom_and_bounds` on the class?